content/stub.completion-ui.js

```diff
@@ -84,15 +84,16 @@ ContactIdentityCompleter.prototype = {
     }
     // and since we can now map from contacts down to identities, map contacts
     //  to the first identity for them that we find...
-    matches = [val.NOUN_ID == Gloda.NOUN_IDENTITY ? val : val.identities[0]
-               for each ([iVal, val] in Iterator(contactToThing))];
+    matches = Array.prototype.map.call(contactToThing, function(val) {
+      return val.NOUN_ID == Gloda.NOUN_IDENTITY ? val : val.identities[0];
+    });
```

Comment:
Meaning: `matches = [... for ([iVal, val] of entries(contactToThing))]`. I should really submit a patch to have this exposed so that we don't maintain a copy of it in the Conversations tree.